index.html

```diff
@@ -238,6 +288,7 @@ <h2 class="first-text-contact">Contact me</h2>
 
 
             </section>
-   
+        
```

Comment:
Kyndly stick to the figma design as much as posible, please notice in the contact form the form does not take to much with of the page.

## Figma design 
![image](https://user-images.githubusercontent.com/95243996/201200324-4905c75c-9706-4c3b-9b9c-165c821cbd36.png)

## Your design
![image](https://user-images.githubusercontent.com/95243996/201200462-bce73bfa-6955-4e7e-8c43-80f2006042ce.png)